src/openrct2/scripting/bindings/entity/ScStaff.cpp

```diff
@@ -104,23 +105,112 @@ namespace OpenRCT2::Scripting
         }
     }
 
-    uint8_t ScStaff::costume_get() const
+    static const DukEnumMap<PeepSpriteType> availableHandymanCostumes({
+        { "handyman", PeepSpriteType::Handyman },
+    });
+
+    static const DukEnumMap<PeepSpriteType> availableMechanicCostumes({
+        { "mechanic", PeepSpriteType::Mechanic },
+    });
+
+    static const DukEnumMap<PeepSpriteType> availableSecurityCostumes({
+        { "security1", PeepSpriteType::Security },
+        { "security2", PeepSpriteType::SecurityAlt },
```

Comment:
More a curiosity question, but what is the difference between these two?

Reply:
I wondered the same thing! One has their hands on the side, and one has their hands on their back. The game alternates between the two.

Reply:
One security guard has his hands next to his body, the other one has his hands on his back.